eng/doc/fips/README.md

```diff
@@ -37,59 +37,100 @@ It is important to note that an application built with Microsoft's Go toolchain
 
 The `GOEXPERIMENT` environment variable is used at build time to select a cryptographic library backend. This modifies the Go runtime included in the program to use the specified platform-provided cryptographic library whenever it calls a Go standard library crypto API. The `GOEXPERIMENT` values that pick a crypto backend are:
 
+- *Recommended:* `systemcrypto` automatically selects the crypto backend we suggest for the target platform
```

Comment:
```suggestion
- *Recommended:* `systemcrypto` automatically selects the suggested crypto backend for the target platform
```

Reply:
Do we ever explicitly state that `systemcrypto` selects OpenSSL for Linux, CNG for windows, etc ... We give a few examples but there isn't really a full table that says what combo of values selects which crypto libraries. I was wondering if we should have something like 

|Target Platform| systemcrypto mapping | library |
| -- | -- | --| 
| Linux | opensslcrypto | OpenSSL |
| Windows | cngcrypto | Windows |

It's fairly implicit though so not sure if it's needed

Reply:
Hmm, yeah, the "we suggest" was meant to point to the Microsoft fork/project doing the suggesting (rather than upstream Go, which suggests using none of these 😄) but this isn't a great way to convey that. I like the idea of trying out a table. I think it would be good to mention that the mapping matches the internal crypto library usage policy requirements for Go. I'm thinking of even dropping the somewhat vague "recommend" term, which I think raises questions about why it's a recommendation (and is it *just* a recommendation?).